changelog.txt

```diff
@@ -26,6 +26,53 @@ Do not use older non-release specific "stable", "staging" or "beta" repositories
 
 *** CHANGELOG ***
 
+2.6.0: release-2.6 2023-06-XX
+
+Highlights:
+* Rule Manager: assign rulelists per Suricata instrument using the new "Tags" feature
+* SFConfig: Securely configure Sensor management interface IP/authentication/adoption VPN using external media
+* Adoption of Sensors over Adoption VPN (call-home vpn, configurable via sfconfig or manually using fleet tool)
+* Major 3rd party component upgrades (Arkime, Suricata, etc)
+* Bug fixes
+
+See Manual for details the new SFConfig/Adoption VPN feature.
+
+- Bug fixes:
+  [SF-3042] - Sensor Installer: Sensors are installed with non-standard EFI mountpoint
+  [SF-3029] - Fleet Tool: Fix adoption VPN client revocation
+  [SF-2995] - Sensor Installer: Add -updates/-security repos to sensor-installer
+  [SF-2960] - Netflow: Fix netflow invalid timestamp milliseconds
+
+- New features and other improvements:
+  [SF-3096] - Ansible: Disable sfconfig auth/vpn import by default when adopting Sensors
+  [SF-3091] - Recorder Instrument: Update arkime to 4.3.1
+  [SF-3089] - PortDiff Instrument: Update to pscan 0.3.0
+  [SF-3085] - Zeek Instrument: Update Zeek to 5.0.9
+  [SF-3080] - Update suricata to 6.0.12, hyperscan to 5.4.2, libhtp to 0.5.43
+  [SF-3056] - Fleet Tool: Support for generating SSH config
+  [SF-3055] - HelloWorld Instrument: Convert Helloworld-instrument to Python 3.10
+  [SF-3052] - Rule Importer: RuleImporter asynchronous rulelist delivery to multiple sensors simultaneously
```

Comment:
[SF-3052] - Rule Importer: Asynchronous rule list delivery to multiple sensors simultaneously